scripts/prepare.sh

```
@@ -1,21 +1,21 @@
-npm run start --update
-npm run stop
+# npm run start --update # wp-env removed
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)